### PR TITLE
Airwindows supports High Precision Display mode

### PR DESCRIPTION
**Pull request by baconpaul**
AW supports high precision display mode with associated
code changes

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -618,6 +620,14 @@ else()
 endif()
 
 
+# Source Groups
+source_group( "Libraries" REGULAR_EXPRESSION "libs/" )
+source_group( "AirWindows" REGULAR_EXPRESSION "libs/airwindows/" )
```
Comment (TommyC81) [inline CMakeLists.txt:625]: Should this be "Airwindows"?

Comment (mkruselj) [reply CMakeLists.txt:625]: Yeah good point there!

Comment (baconpaul) [reply CMakeLists.txt:625]: Yeah sorry we got the rest of em. Let me add a change to your def and push it over in this PR evil.

Comment (baconpaul) [reply CMakeLists.txt]: OK forced that. Merging now since the code all built on the prior branch.

**Pull request merged**